Cargo.toml

```diff
@@ -9,6 +9,7 @@ authors = ["Chris Holcombe <xfactor973@gmail.com>"]
 categories = ["api-bindings"]
 
 [dependencies]
-blkid-sys = "^0.1"
+blkid-sys = { git = "https://github.com/Silverhorn27/blkid-sys", branch = "feat/update-dep-version" }
```

Comment:
I merged the blkid-sys crate and published it. Would you mind updating this dependency?

Reply:
Thank you a lot.
Yes, sure, I'm going to update this PR. I let you know :)

Reply:
Fixed.